.env

```diff
@@ -2,4 +2,15 @@ REACT_APP_CHAIN_ID="1"
 REACT_APP_NETWORK_URL="https://mainnet.infura.io/v3/1221fd11e90849509afafd330ec7acc6"
 
 # Override with your APP_ID
-REACT_APP_ID="0"
\ No newline at end of file
+REACT_APP_ID="0"
+
+# Domain
+# Accept dev as a list of hosts
+REACT_APP_DOMAIN_PREFIX_DEV=cow-trade.dev
+REACT_APP_DOMAIN_PREFIX_STAGING=cow-trade.staging
+REACT_APP_DOMAIN_PREFIX_PROD=cow.trade
```

Comment:
Although this is correct right now, it'll change before the release.
Just to keep in mind.
Create an issue maybe? :shrug:

Reply:
:+1: